toolchain/pkg.yaml

```diff
@@ -5,7 +5,7 @@ dependencies:
   - stage: binutils
   - stage: gcc
   - stage: linux-headers
-  - stage: musl
+  - stage: toolchain-musl
   - stage: make
   - stage: golang
 # TODO: run this test once we figure out the best way to not let temporary files into the final image
```

Comment:
we can run this now?

Reply:
I mean `bldr` now should discard test step output

Reply:
hmm, makes sense, thanks. will do once reproducibility passes